core/src/contract_env.rs

```diff
@@ -20,20 +19,15 @@ impl ContractEnv {
         }
     }
 
-    pub fn duplicate(&self) -> Self {
+    pub fn __duplicate(&self) -> Self {
```

Comment:
Should not be used by a user directly in a module but is used internally outside the core crate.